static/app/components/compactSelect/styles.tsx

```diff
@@ -134,3 +138,16 @@ export const CheckWrap = styled('div')<{isSelected: boolean; multiple: boolean}>
   padding-bottom: 1px;
   pointer-events: none;
 `;
+
+export const EmptyMessage = styled('p')`
+  text-align: center;
+  color: ${p => p.theme.subText};
+  padding: ${space(1)} ${space(1.5)} ${space(1.5)};
+  margin: 0;
+
+  /* Message should only be displayed when _all_ preceding lists are empty */
+  display: block;
+  ul:not(:empty) ~ & {
+    display: none;
+  }
```

Comment:
The neat thing about this is that it's all CSS-based — no need to use JS to check if the lists are empty.